src/v2/guide/routing.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Routing
+title: Định tuyến
 type: guide
 order: 501
 ---
 
-## Official Router
+## Thư viện Chính thức
 
-For most Single Page Applications, it's recommended to use the officially-supported [vue-router library](https://github.com/vuejs/vue-router). For more details, see vue-router's [documentation](https://router.vuejs.org/).
+[vue-router](https://github.com/vuejs/vue-router) là thư viện chính thức của [Vue](https://github.com/vuejs) và nó được khuyến nghị sử dụng cho phần lớn các Ứng dụng Đơn Trang (Single Page Applications). Để biết chi tiết, hãy xem [tài liệu chính thức](https://router.vuejs.org/) của vue-router.
```

Comment:
Chỉ nên nói là "ứng dụng một trang". Ngoài ra "khuyến nghị" nghe không tự nhiên bằng "khuyến khích." Toàn bộ câu này, nếu là mình thì sẽ tách ra làm hai, như sau:

> Vue-router thư viện routing (định tuyến) chính thức của Vue. Chúng tôi khuyến khích sử dụng nó cho phần lớn các ứng dụng một trang (Single-Paged Applications). Để biết thêm chi tiết, hãy xem tài liệu chính thức của vue-router.